roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -15,7 +15,7 @@ auth service required = none
 auth client required = none
 auth supported = none
 {% endif %}
-fsid = {{ fsid }}
+{% if not mon_containerized_deployment_with_kv%}fsid = {{ fsid }}{% endif %}
```

Comment:
There is no need to do anything in the ceph-common role since we run containers. Common is not a dependency of monitor role when we run in containers.